pkg/sql/catalog/table_elements.go

```diff
@@ -440,6 +440,9 @@ type Constraint interface {
 	// GetName returns the name of this constraint update mutation.
 	GetName() string
 
+	// GetConstraintType returns the constraint type.
+	GetConstraintType() descpb.ConstraintType
```

Comment:
Rather than make this a method on the `Constraint` interface, can you make this a function instead? This isn't terribly important in this case, but a useful general principle is to keep the set of object methods to a minimum and to keep separate any logic which can be expressed using method calls and doesn't need any state that's internal to the object.